runtime/compiler/control/J9Options.cpp

```diff
@@ -128,6 +128,8 @@ int64_t J9::Options::_oldAgeUnderLowMemory = 1000*60*5; // 5 minute
 int64_t J9::Options::_timeBetweenPurges = 1000*60*1; // 1 minute
 bool J9::Options::_shareROMClasses = false;
 int32_t J9::Options::_sharedROMClassCacheNumPartitions = 16;
+int32_t J9::Options::_highActiveThreadThreshold = -1;
+int32_t J9::Options::_veryHighActiveThreadThreshold = getNumUsableCompilationThreads() * 0.9;
```

Comment:
This will not work because getNumUsableCompilationThreads() does not have a proper value before initialization of options starts. We need to do use the same scheme as for `J9::Options::_highActiveThreadThreshold`